sasjs-tests/public/config.json

```diff
@@ -3,10 +3,10 @@
   "password": "",
   "sasJsConfig": {
     "serverUrl": "",
-    "appLoc": "/Public/app/adapter-tests",
+    "appLoc": "/Public/app/adapter-tests/services",
```

Comment:
is this correct?  normally "services" would be left out of the appLoc

Reply:
yes, this is correct. In this case, we need services. I tested, it wasn't working when services was missing